components/bl-backdrop-component/src/loaders/DotedCircleLoader.js

```diff
@@ -0,0 +1,15 @@
+export function DotedCircleLoader() {
```

Comment:
we are using kebab-case for file naming

`bar-loader.js`

Also, if it in `loaders` directory, i think loader in file name in redundant

so `bar.js` `doted-circle.js` ...